atlasdb-cassandra-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraService.java

```diff
@@ -65,4 +70,32 @@ public static synchronized void stop() {
         daemon.deactivate();
         daemon = null;
     }
+
+    private static void waitForDaemonToStart() throws InterruptedException {
```

Comment:
nit: probably want to use try-with-resources to make sure we close the socket if we fail.

```
    // TODO load host/port from selected config?
    int port = 9160;
    InetSocketAddress endPoint = new InetSocketAddress(InetAddress.getLocalHost(), port);
    while (iterationsToCheck-- > 0) {
        try (Socket socket = new Socket()) {
            socket.connect(endPoint, 1000);
            return;
        } catch(IOException e) {
            try {
                Thread.sleep(1000);
            } catch (InterruptedException ignored) { /* fine */ }
        }
    }
```